operations/restrictions.md

```diff
@@ -395,6 +395,12 @@ This reduces the overall number of restrictions and improves system performance.
    1) If the new interval overlaps the old interval, the old restriction will be spliced before and after the new interval. Restrictions matching the old restriction are then added at the appropriate interval along with the new restriction.
    2) If the new interval does _not_ overlap the old interval, the new restriction is added as usual.
 
+### Affected restrictions
+
+Merging algorithm will also replace _all_ exceptions with exceptions defined in the request. It will also remove exception if null is sent. This can potentially remove exceptions defined in Mews but not supported in other applications.
```

Comment:
```suggestion
```
I suggest deleting this paragraph altogether, because it merely explains what _would_ have happened if the following paragraph was not true, therefore it's simply irrelevant and misleading.